### ISSUE TITLE
More glitchy graphics

### ISSUE THREAD

**Issue by nippip** [codecombat/codecombat#1933]
I have a Macbook pro running 10.6.8 with 4GB and Intel Core i7
When viewing codecombat with an updated version of Chrome (ver 39.0.2121.95) the graphics are a mess in the warrior window as shown below.
The following demo works just fine although a bit slow. http://madebyevan.com/webgl-water/

![screen shot 2014-12-13 at 9 03 23 pm](https://cloud.githubusercontent.com/assets/2540198/5426422/46385740-830d-11e4-8e02-06623713dcd4.png)

**Comment by mgorbet** [comment]
We are having the same problem - Chrome on a macbook pro early 2008 with OSX10.6.8.

![image](https://cloud.githubusercontent.com/assets/293459/5691567/c0203278-98cd-11e4-8a57-5ed73b542c4f.png)

On Safari on that same system, it didn't load at all.  (halfway through the loading bar it hung).

Will try FF.   

To me, it looks like a graphics memory access issue - the repeated tiles are other graphics memory that has been assigned wrong.  Not sure why this would happen but it's a pretty low-level bug.

The gameplay works fine - it's just the display makes it impossible to play.

**Comment by nwinter** [comment]
So it sounds like both problems on Mac have been on 10.6.8 so far. Is it possible to try updating to a more recent OS X version that might have better graphics drivers?

**Comment by mgorbet** [comment]
Could be... I tried with FF - had to download the latest version.... and it worked.  So that's cool for now.  I spent today upgrading the Macbook pro so it is ready for a later OS (it needed more ram and a bigger disk) and my next step is to upgrade to Mavericks.  I'll let you know if that solves it, in any event.

By the way, my 9 yo son LOVES the game - he went through the whole dungeon in a couple of hours this afternoon, and is thrilled to have IF statements at his disposal now.  Congrats on this.
